guide/app_ssystem_ini.tex

```diff
@@ -18,16 +18,15 @@ \section{Solar System Configuration Files}
 The minor bodies can be modified by placing a privately modified 
 copy of \file{ssystem\_minor.ini} into your own
 \file{data/} directory (see Chapter~\ref{sec:FilesAndDirectories}). 
-You can edit the file either manually, or with the
-Solar System Editor plugin (see
-Section~\ref{sec:plugins:SolarSystemEditor}).
+You can edit the file either manually, or with the Solar System Editor plugin (see
+Section~\ref{sec:plugins:SolarSystemEditor}). The private user file is automatically created by the Solar System Editor plugin.
 
 
 %Deprecated parameters are marked by gray background. 
 %Possible new parameters are marked by yellow background.
 %% GZ This is no longer true. Reactivate somehow? 
 
-Each object's data are described in its own section which is typically
+Each object's data is described in its own section which is typically
```

Comment:
data can be used with singular or plural verb. We definitely have more than one value, so the plural form "are" appears more natural to use.